src/OutputWriters/netcdf_output_writer.jl

```diff
@@ -149,7 +149,8 @@ Keyword arguments
 - `array_type`: The array type to which output arrays are converted to prior to saving.
                 Default: Array{Float32}.
 
-- `field_slicer`: An object for slicing field output in ``(x, y, z)``, including omitting halos.
+- `field_slicer`: An object for slicing field output in ``(x, y, z)``, including omitting halos,
+                  which can be done with the keyword `with_halos`.
```

Comment:
Thanks, good catch! I noticed the `JLD2OutputWriter` has a better explanation so I copied it into this suggestion:

```suggestion
- `field_slicer`: An object for slicing field output in ``(x, y, z)``, including omitting halos.
                  Has no effect on output that is not a field. `field_slicer = nothing` means
                  no slicing occurs, so that all field data, including halo regions, is saved.
                  Default: `FieldSlicer()`, which slices halo regions.
```

Reply:
Doesn't that repeat what's already on the following lines? Also, it makes no mention of the `with_halos` keyword, which I think would be nice, no?

Reply:
Ah sorry woops! I only saw the changes without paying attention to the rest of the file haha.